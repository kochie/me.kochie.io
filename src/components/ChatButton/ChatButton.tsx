import React from "react";

import { button } from "./chat-button.css";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";

export default () => {
  return (
    <div className={button}>
      <FontAwesomeIcon icon="comment" />
    </div>
  );
};
