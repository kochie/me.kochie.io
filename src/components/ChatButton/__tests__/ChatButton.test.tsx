import renderer from "react-test-renderer";
import React from "react";

import ChatButton from "../ChatButton";

it("by me a coffee renders correctly", () => {
  const tree = renderer.create(<ChatButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
