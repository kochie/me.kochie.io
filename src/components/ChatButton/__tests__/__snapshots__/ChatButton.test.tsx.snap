// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`by me a coffee renders correctly 1`] = `
<div
  className="button"
/>
`;
